src/git.js

```diff
@@ -65,6 +65,29 @@ module.exports = function gitFactory(repository = '', origin = 'origin') {
                 .then( branches => branches.all);
         },
 
+        /**
+         * Check if the release branch for master exists or suggest main instead.
+         * @param {String} branchName - the branch name
+         * @returns {Promise}
+         */
+        hasReleaseBranch(branchName) {
```

Comment:
suggestion: As this returns a string instead of a boolean, `getReleaseBranch()` or `getReleaseBranchName()` may be a better name